src/lib/style/base/_core.scss

```diff
@@ -1236,25 +1236,42 @@ hr ~ .SRC-wrapper {
 main,
 .modal,
 #rootPanel {
-  a:not(.btn),
-  .bootstrap-4-backport a:not(.btn),
-  .markdown td a:not(.btn),
-  .markdown a.SRC-userCard:not(.btn),
-  .secondary-links a:not(.btn),
-  .markdown .secondary-links a:not(.btn) {
-    @include SRC_Mixins.normalLink;
-    &:visited,
-    &:active,
-    &:link {
+  a,
+  .bootstrap-4-backport a,
+  .markdown td a,
+  .markdown a.SRC-userCard,
+  .secondary-links a,
+  .markdown .secondary-links a,
+  .modal-body .dropdown-menu a {
+    &:not(.btn) {
       @include SRC_Mixins.normalLink;
+      &:visited,
+      &:active,
+      &:link {
+        @include SRC_Mixins.normalLink;
+      }
+    }
+  }
+
+  .modal-body .dropdown-menu a {
```

Comment:
Now that we style links in modal-body, this messes up anchors that are being used as dropdown menu items!  This is to correct that issue.